site-src/geps/gep-1324.md

```diff
@@ -0,0 +1,127 @@
+# GEP-1324: Service Mesh in Gateway API
+
+* Issue: [#1324](https://github.com/kubernetes-sigs/gateway-api/issues/1324)
+* Status: Provisional
+
+## Overview
+
+Gateway API represents the next generation of traffic routing APIs in Kubernetes. While most of the current work for Gateway API is focused on the ingress use-case, support for service mesh implementations within the spec would be advantageous to the greater community. Therefore, this GEP serves as the genesis for developing common patterns for using Gateway API for east/west traffic within a mesh implementation.
+
+## Goals
+
+* Augment the Gateway API specification with the necessary models, resources, and modifications so that the xRoute primitives are useable by service mesh implementations
```

Comment:
```suggestion
* Augment the Gateway API specification with the necessary models, resources, and modifications so that the xRoute primitives are usable by service mesh implementations
```